.github/workflows/ci-unix-static-av2.yml

```diff
@@ -39,6 +39,10 @@ jobs:
         gcc-version: ${{ matrix.gcc }}
         libyuv: 'LOCAL'
 
+    - name: Install cpuinfo
+      run:
+        sudo apt install -y cpuinfo
```

Comment:
Why do we need to install cpuinfo here? It would be good to mention this in the commit message.

Reply:
This is not needed anymore, removing